src/coreclr/dlls/mscordbi/CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ set(COREDBI_LIBRARIES
     mdruntimerw-dbi
     mddatasource_dbi
     corguids
+    minipal_objects
```

Comment:
Why not linking `minipal`?

Reply:
Seems like a mistake to me. Will change.